integration-test-groups/jdbc/db2/src/main/resources/application.properties

```diff
@@ -0,0 +1,44 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+
+#
+# Quarkus :: DS
+#
+quarkus.datasource.postgresql.db-kind=postgresql
+quarkus.datasource.postgresql.jdbc.max-size=8
+
+quarkus.datasource.mysql.db-kind=mysql
+quarkus.datasource.mysql.jdbc.max-size=8
+
+quarkus.datasource.mariadb.db-kind=mariadb
+quarkus.datasource.mariadb.jdbc.max-size=8
+
+quarkus.datasource.oracle.db-kind=oracle
+quarkus.datasource.oracle.jdbc.max-size=8
+
+quarkus.datasource.h2.db-kind=h2
+quarkus.datasource.h2.jdbc.max-size=8
+
+quarkus.datasource.db2.db-kind=db2
+quarkus.datasource.db2.jdbc.max-size=8
+
+quarkus.datasource.mssql.db-kind=mssql
+quarkus.datasource.mssql.jdbc.max-size=8
+
```

Comment:
Can we only keep `quarkus.datasource.db2` configurations sine it is for `db2` testing?

Reply:
@zhfeng I thought that the `application.properties` file should be the same for all modules, otherwise group tests don't work properly.